web-common/src/components/menu-v2/MenuItem.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  import { MenuItem } from "@rgossiaux/svelte-headlessui";
+
+  export let as: "a" | "button" = "a";
```

Comment:
Would be good to have some documentation around these

Reply:
Actually I wasn't using this prop anywhere, so just removed it.